release-tools/prow.sh

```diff
@@ -37,6 +37,8 @@
 # - kind (https://github.com/kubernetes-sigs/kind) installed
 # - optional: Go already installed
 
+set -x
```

Comment:
We can't make changes under release-tools in the sidecar repo directly.

Reply:
Sorry I am not familiar with this part, so does it mean I need to make changes in https://github.com/kubernetes-csi/csi-release-tools？Will go through the detail docs, thanks for the info!